src/EditorFeatures/Test/EditorServicesTest.csproj

```diff
@@ -361,4 +362,4 @@
     <Import Project="..\..\..\build\Roslyn.Toolsets.Xunit.targets" />
     <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Did VS eat the line ending?  It's generally nice to have a trailing newline for git tools.